flink-connectors/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/internals/KafkaTopicsDescriptor.java

```diff
@@ -41,17 +42,25 @@
     private final Pattern topicPattern;
 
     public KafkaTopicsDescriptor(
-            @Nullable List<String> fixedTopics, @Nullable Pattern topicPattern) {
+        @Nullable List<String> fixedTopics, @Nullable Pattern topicPattern) {
         checkArgument(
-                (fixedTopics != null && topicPattern == null)
-                        || (fixedTopics == null && topicPattern != null),
-                "Exactly one of either fixedTopics or topicPattern must be specified.");
+            (fixedTopics != null && topicPattern == null)
+                || (fixedTopics == null && topicPattern != null),
+            "Exactly one of either fixedTopics or topicPattern must be specified.");
 
         if (fixedTopics != null) {
             checkArgument(
-                    !fixedTopics.isEmpty(),
-                    "If subscribing to a fixed topics list, the supplied list cannot be empty.");
+                !fixedTopics.isEmpty(),
+                "If subscribing to a fixed topics list, the supplied list cannot be empty.");
+            fixedTopics.forEach(topic -> {
+                checkNotNull(topic, "An null topic exists in the subscribed topics list.");
```

Comment:
Here, we can use 
'
checkArgument(
                !org.apache.flink.util.StringUtils.isNullOrWhitespaceOnly(topic), "topic in the subscribed topics list cannot be null or empty string.");
'
to check